packages/neuron-wallet/src/utils/createLoopTask.ts

```diff
@@ -0,0 +1,37 @@
+import { BrowserWindow } from 'electron'
+import env from '../env'
+import SyncBlocksService from '../services/syncBlocks'
+import initConnection from '../typeorm'
+
+// create a background task to sync transactions
+// this task is a renderer process
+const createLoopTask = () => {
+  let loopWindow: BrowserWindow | null = new BrowserWindow({
+    x: 10,
+    y: 10,
+    show: false,
+    webPreferences: {
+      nodeIntegration: false,
```

Comment:
use `nodeIntegrationInWorker` instead of `nodeIntegration` according to https://electronjs.org/docs/tutorial/multithreading#multi-threaded-nodejs, as required in the past, the `nodeIntegration` should be `false`